src/auth/MasterKey.ts

```diff
@@ -0,0 +1,74 @@
+import ExtendableError from 'es6-error'
+import {useMutation, UseMutationOptions} from 'react-query'
+
+import {decryptData, encryptData} from '../legacy/commonUtils'
+import storage from '../legacy/storage'
+import {YoroiWallet} from '../yoroi-wallets'
+
+export class CredentialsNotFound extends ExtendableError {}
+
+export const useRevealMasterKey = (
```

Comment:
id like to avoid the use of the term "master"
"main" or "root"?

Reply:
if a hook returns a value, prefer useQuery()

Reply:
> id like to avoid the use of the term "master" "main" or "root"?

Moving to `root`

Reply:
> if a hook returns a value, prefer useQuery()

🤦